packages/style/spec/style-spec.js

```diff
@@ -0,0 +1,79 @@
+'use babel'
+
+/* @flow */
+
+import styler from '../lib/index'
+import { it } from 'jasmine-fix'
+import React from 'react'
+import { mount } from 'enzyme'
+
+const style = styler()
+
+@style
```

Comment:
Any ideas you're doing `styler()` and then `@style` instead of just `@styler()`?